### DISCUSSION TITLE
bug on zirnox on this new version x5057

### DISCUSSION THREAD

**Discussion by idalx** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1638] [General]
the counters inside the zirnox are not showing water and carbon dioxide values 
![Captura de Tela (284)](https://github.com/user-attachments/assets/b5ceb477-a4d7-4856-a976-49bdcf213a92)
![Captura de Tela (285)](https://github.com/user-attachments/assets/23b9e51f-6a72-42fa-9701-57ee7ca62cf7)
![Captura de Tela (286)](https://github.com/user-attachments/assets/20ec8800-9268-4632-8acd-07ae1e338845)

**Comment by thePalindrome** [comment]
It also applies to the Oil Derrick, but *not* to the Oil Refinery

**Comment by HbmMods** [comment]
Should have been fixed with yesterday's version, although the pumpjack supposedly still has some weirdness going on with the gauges
